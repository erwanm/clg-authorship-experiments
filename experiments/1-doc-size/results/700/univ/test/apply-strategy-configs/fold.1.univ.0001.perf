0.539000	0.770000	0.700000
