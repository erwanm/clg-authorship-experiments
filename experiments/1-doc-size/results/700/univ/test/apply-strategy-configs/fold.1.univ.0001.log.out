apply-config.sh: applying config '/tmp/700/process/selected-meta-configs/fold.1.strategy-as-meta.univ.0001.model/strategy-configs/fold.1.univ.0001' to cases '/home/moreaue/dump/expe1//results/700/univ/test/apply-strategy-configs/only-cases.tmp.fZEAOURN'; results written to '/home/moreaue/dump/expe1//results/700/univ/test/apply-strategy-configs/fold.1.univ.0001'
apply-config.sh: done.
